src/contradictory_claims/__init__.py

```diff
@@ -3,5 +3,6 @@
 """A package for finding contradictory claims related to COVID-19 drug treatments in the CORD-19 literature."""
 
 from .data.make_dataset import *  # noqa:F401,F403
+# from .extract_claims import  # noqa:F401,F403
```

Comment:
I have not included this, because it is messing with one of the imports from other modules, do check the way I have used in the test, it works even if you won't put it in  __init__ , not sure what the best practice is

Reply:
Okay, we can check on this and add it back in later. I think we'll eventually want to include this import for packaging vanity basically